adapters/copy.go

```diff
@@ -0,0 +1,29 @@
+package adapters
+
+import (
+	"github.com/smartcontractkit/chainlink/store"
+	"github.com/smartcontractkit/chainlink/store/models"
+	"github.com/tidwall/gjson"
+)
+
+// Copy obj keys refers to which value to copy inside `data`,
+// each obj value refers to where to copy the value to inside `data`
```

Comment:
Let's place an example in the documentation:

```
Copy is a json object whose keys dictate the source and whose values are the destination of a copy operation.
`{"src1": "dst1"}`
```

On that note, how do we handle a situation where we the destination is more than a string?

`{"src": {"complicated": "org"}}`

Looks like we would `String()` it and make it a rather sophisticated key.

Reply:
With docs, you mean this page?
https://github.com/smartcontractkit/chainlink/wiki/Adapters

I'd add some in, EthInt256 is missing also, don't have the perms though.

Reply:
I can update the wiki documentation.